Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   isCustomProfile: 0
   mixedRealityControllerMappingProfiles:
   - controllerType:
-      reference: Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.WindowsMixedRealityController,
+      reference: Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityController,
         Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality
```

Comment:
nit: this still feels pretty long. What was the incentive for adding input to wmr specific items?

Reply:
To match the top level feature namespace

---
In reply to: [267120422](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267120422) [](ancestors = 267120422)